src/coreclr/inc/clrconfigvalues.h

```diff
@@ -754,7 +754,7 @@ RETAIL_CONFIG_DWORD_INFO(EXTERNAL_EnableAVX512CD,     W("EnableAVX512CD"),     1
 RETAIL_CONFIG_DWORD_INFO(EXTERNAL_EnableAVX512CD_VL,  W("EnableAVX512CD_VL"),  1, "Allows AVX512CD_VL+ hardware intrinsics to be disabled")
 RETAIL_CONFIG_DWORD_INFO(EXTERNAL_EnableAVX512DQ,     W("EnableAVX512DQ"),     1, "Allows AVX512DQ+ hardware intrinsics to be disabled")
 RETAIL_CONFIG_DWORD_INFO(EXTERNAL_EnableAVX512DQ_VL,  W("EnableAVX512DQ_VL"),  1, "Allows AVX512DQ_VL+ hardware intrinsics to be disabled")
-RETAIL_CONFIG_DWORD_INFO(EXTERNAL_EnableAVX512F,      W("EnableAVX512F"),      1, "Allows AVX512F+ hardware intrinsics to be disabled")
```

Comment:
This isn't going to change the SPMI numbers because this is a VM flag, so the ISAs reported as supported from the SPMI VM cache won't change. It does change the results for real runs, however.

If you want to see the results in SPMI, you need an additional JIT side tweak; as elaborated on here: https://github.com/dotnet/runtime/pull/83648#issuecomment-1482951574

The plan would be for me to do the work to allow the JIT versions of these flags (defined here: https://github.com/dotnet/runtime/blob/main/src/coreclr/jit/jitconfigvalues.h#L321-L328) to not only be used by the AltJIT to enable otherwise unsupported ISAs, but to also be used by scenarios like SPMI to disable ISAs.

Reply:
Right, I realized this based on the other issue conversation. I do like the idea of exposing the flags for SPMI usage.